schemas/app/inventory/StockTransfer.json

```diff
@@ -65,6 +65,25 @@
       "abstract": true,
       "fieldname": "backReference",
       "section": "References"
+    },
+    {
+      "fieldname": "isReturn",
+      "label": "Is Return",
+      "placeholder": "Is Return",
+      "fieldtype": "Check",
+      "default": false,
+      "section": "Return"
+    },
+    {
+      "abstract": true,
+      "fieldname": "returnAgainst",
+      "section": "Return"
+    },
+    {
+      "fieldname": "isItemsReturned",
```

Comment:
bad grammar, just call it `isCompletelyReturned` (or even `isReturned` considering that `isReturn` will be removed)